src/networks.js

```diff
@@ -127,16 +127,16 @@ var networks = {
   }
 }
 
-function estimateFee(type) {
-  return function(tx) {
+function estimateFee (type) {
+  return function (tx) {
     var network = networks[type]
     var baseFee = network.feePerKb
     var byteSize = tx.toBuffer().length
 
     var fee = baseFee * Math.ceil(byteSize / 1000)
-    if (network.dustSoftThreshold == undefined) return fee
+    if (network.dustSoftThreshold === undefined) return fee
```

Comment:
Sure it's unlikely that someone would explicitly set it to `null` on purpose, I was just pointing out that this switch here changes the logic. I'm happy to leave it as is and update it when someone actually has a problem with it.